cmd/ci-operator/main.go

```diff
@@ -359,6 +361,9 @@ func bindOptions(flag *flag.FlagSet) *options {
 	flag.StringVar(&opt.variant, "variant", "", "Variant of the project's ci-operator config (used by configresolver)")
 
 	flag.String("kubeconfig", "", "Legecay flag kept for compatibility reasons. Doesn't do anything.")
+	// SA ci-operator on app.ci will be used to generate the kubeconfig file
```

Comment:
For now it will be on api.ci or not?

Reply:
For now, this flag is not be used.
This PR wont have any impact for any promotion job.

My next step:
Move one promotion job to build01 (by providing such kubeconfig) to see if it works (push image from build01 to app.ci). 
If it goes through, promotion jobs are ready for migration.

(We still need to mirror the whole existing images from api.ci to app.ci. But that is not in the scope of the current PR.)

Reply:
My point is, that comment is wrong. Initially, this will be a SA on api.ci, later one on app.ci. I don't think we should reference either as comment